core/src/main/java/bio/terra/pearl/core/service/notification/substitutors/EnrolleeEmailSubstitutor.java

```diff
@@ -177,20 +196,23 @@ public String getParticipantSupportEmailLink(PortalEnvironment portalEnvironment
     }
 
     /**
-     * gets a link the participant can use to create their b2c account, given that they already exist in Juniper
+     * gets a b2c redirect link; e.g., invitation or reset password link. Path must be from
```

Comment:
I'm not sure I'm following what's happening with this method--can you refine the comment?